package.json

```diff
@@ -41,29 +47,26 @@
     "eslint-loader": "^4.0.2",
     "eslint-plugin-cypress": "^2.11.3",
     "eslint-plugin-no-only-tests": "^2.6.0",
-    "ignore-emit-webpack-plugin": "^2.0.3",
+    "eslint-plugin-storybook": "^0.6.15",
     "ignore-styles": "^5.0.1",
     "jsdom": "^16.4.0",
     "jsdom-global": "^3.0.2",
     "less": "^3.12.2",
     "less-loader": "^5.0.0",
-    "less-plugin-inline-svg": "^2.0.1",
-    "mini-css-extract-plugin": "^0.9.0",
     "mocha": "^7.2.0",
     "nyc": "^15.1.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
According to https://github.com/wikimedia/wikipedia-preview/pull/177/commits/f1e0aa2b863d07ae795ce3bc42607b4e8cc7e800 it looks like react was added as a consequence of the storybook update. @hueitan can you confirm this is fine and we should leave as dependency?

Reply:
removed, we don't write jsx form in the storybook